backend/dissemination/api/db_views.sql

```diff
@@ -0,0 +1,67 @@
+
+begin;
+
+drop view if exists api.vw_general;
```

Comment:
It doesn't have to be in this PR, but I _think_ we want to get rid of the `vw` in our views.

That is, the end-user of the API doesn't care that this is a view. Are we able to do that, or do we end up with name conflicts?

Actually, ignore that comment. We're probably going to have to do a small refactor on this anyway, as we may need to use a different schema. That is, we need to be able to version the API, and we do that by having multiple schema with multiple APIs live at the same time. In this regard, we may want an api_v1_0_0 schema. 

So, for now, ignore this comment, but it is a "thought placeholder" for the future. I'll get the API versioning doc into an ADR format for consideration, and that will help with documenting/ticketing how we need to handle the API schemas.

(It will be a small refactor, I think, hence it doesn't *need* to be picked up right now.)